ui/pages/confirmation/util.ts

```diff
@@ -41,13 +43,22 @@ export function processError(
  *
  * @param input - The message or component to process.
  * @param fallback - The fallback string to use when the input is not valid.
+ * @param translate - Optional function to translate the input string.
  * @returns The processed message.
  */
 export function processString(
   input: undefined | string | ResultComponent | ResultComponent[],
   fallback: string,
+  translate?: TranslateFunction,
```

Comment:
As mentioned in the `MetaMaskController`, we can translate in the background before we call the `success` method meaning we shouldn't need this translation support here as the burden is currently on the caller to translate.

Reply:
Great call. removed!